src/extensions/main-api/power.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { powerMonitor } from "electron";
+import type { Disposer } from "../../common/utils/disposer";
+type PowerEventListener = () => void;
```

Comment:
Please export this type since it will be required to soon anyway for documentation purporses

Reply:
Done.